docs/analysis-workflows/analysis-overview.md

```diff
@@ -0,0 +1,21 @@
+---
+id: analysis-overview
+title: Harmonization Overview
+sidebar_label: Overview
+---
+
+Data submitted to the the ARGO Data Platform will undergo unified molecular analysis through a harmonized pipeline. Consistent pipelien analysis ensures that data can be considered functionally equvalent across different data sets, including all ARGO Member programs as well as any other initiatives that have adopted the ARGO harmonization pipelines.
+
+ICGC ARGO will accept a range of datatypes including:
+
+- DNA and RNA samples from both Whole Exome Seequencing (WXS) and Whole Genome Sequencing (WGS)
+- DNA Methylation data
+- Transcriptomic data
+- Proteomic data
+- Slide Images
+
+Pipelines, and the individual analysis workflows that they are constructed from, have been devloped by the ICGC bioinformatics team based on established and best community practices.

```

Comment:
typo `devloped`
should it be "ICGC ARGO" bioinformatics team? or DCC?

Reply:
Well, the DCC is larger... i thought fo accuracy it would be best to state that "bioinformatics" was involved.  I will say `DCC Bioinformatics`

Reply:
yeah no problem, just was wondering if they are part of "DCC"